src/site/xdoc/downloads.xml

```diff
@@ -70,26 +70,26 @@ under the License.
     </tr>
     <tr>
       <td style="test-align: left">
-        2.5.2
+        2.5.3
       </td>
       <td style="test-align: left">
         2022/12/03
```

Comment:
Please also change the release date

Reply:
thanks, I updated it to 2023/02/05 (my time zone is PST and should be still within Feb 05 )